<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1048</width>
    <height>777</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RESTFUL Debuger</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1041</width>
      <height>701</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="layAddress">
       <item>
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>URI</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="txtUri">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;http://ip.jsontest.com/&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="placeholderText">
          <string>http://youruri.com</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnExecute">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Execute</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <layout class="QVBoxLayout" name="layRequestContent">
       <item>
        <widget class="QTabWidget" name="tabContent">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>209</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="tabContentBody">
          <attribute name="title">
           <string>Content</string>
          </attribute>
          <widget class="QWidget" name="verticalLayoutWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>10</y>
             <width>1021</width>
             <height>161</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="layContentBody">
            <item>
             <widget class="QPlainTextEdit" name="txtRequestContent">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="tabContentAuthorization">
          <attribute name="title">
           <string>Authorization</string>
          </attribute>
          <widget class="QWidget" name="formLayoutWidget">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1011</width>
             <height>171</height>
            </rect>
           </property>
           <layout class="QFormLayout" name="layContentAuthorization">
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Auth Type</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="cbxAuth">
              <item>
               <property name="text">
                <string>None</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Simple</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Basic</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>OAuth</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Username</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Password</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLineEdit" name="txtAuthUsername"/>
              </item>
              <item>
               <widget class="QLineEdit" name="txtAuthUsernameOAuth"/>
              </item>
             </layout>
            </item>
            <item row="4" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLineEdit" name="txtAuthPassword">
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txtAuthPasswordOAuth">
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="tabContentProxy">
          <attribute name="title">
           <string>Proxy</string>
          </attribute>
          <widget class="QWidget" name="formLayoutWidget_2">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>661</width>
             <height>176</height>
            </rect>
           </property>
           <layout class="QFormLayout" name="layContentProxy">
            <item row="0" column="0">
             <widget class="QCheckBox" name="chkProxyEnabled">
              <property name="text">
               <string>CheckBox</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Address</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Port</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Username</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Password</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="txtProxyAddress"/>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="txtProxyUsername"/>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="txtProxyPassword">
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="spnProxyPort">
              <property name="maximum">
               <number>999999</number>
              </property>
              <property name="value">
               <number>8080</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="tabContentHeader">
          <attribute name="title">
           <string>Header</string>
          </attribute>
          <widget class="QWidget" name="verticalLayoutWidget_2">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>0</y>
             <width>701</width>
             <height>161</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="layContentHeader">
            <item>
             <widget class="QTextEdit" name="txtContentHeader"/>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="layMethodSelector">
       <item>
        <widget class="QRadioButton" name="rdbGet">
         <property name="text">
          <string>&amp;GET</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rdbPost">
         <property name="text">
          <string>POST</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rdbPut">
         <property name="text">
          <string>PUT</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rdbPatch">
         <property name="text">
          <string>PATCH</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="rdbdelete">
         <property name="text">
          <string>DELETE</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <layout class="QVBoxLayout" name="layResponseStatus">
       <item>
        <widget class="QLabel" name="lbl_status">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0">
      <layout class="QHBoxLayout" name="layResult">
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_header">
          <attribute name="title">
           <string>Header</string>
          </attribute>
          <widget class="QWidget" name="horizontalLayoutWidget_6">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>1001</width>
             <height>191</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="layResultHeader">
            <item>
             <widget class="QPlainTextEdit" name="txtResultHeader">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
         <widget class="QWidget" name="tab_body">
          <attribute name="title">
           <string>Body</string>
          </attribute>
          <widget class="QWidget" name="horizontalLayoutWidget_5">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>0</y>
             <width>981</width>
             <height>181</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="layResultBody">
            <item>
             <widget class="QPlainTextEdit" name="txtResultBody">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1048</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSair">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actExit"/>
   </widget>
   <addaction name="menuSair"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
